x-pack/plugins/inference/server/chat_complete/adapters/get_inference_adapter.ts

```diff
@@ -0,0 +1,29 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { InferenceConnectorType } from '../../../common/connectors';
+import type { InferenceConnectorAdapter } from '../types';
+import { openAIAdapter } from './openai';
+
+export const getInferenceAdapter = (
+  connectorType: string
```

Comment:
Can this type be narrowed?

```suggestion
  connectorType: InferenceConnectorType
```

Reply:
https://github.com/elastic/kibana/pull/191069/commits/a8d38ea287ac0a38ac0154a191599eac1cb33230